end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Load_Tests.cy.js

```diff
@@ -1,98 +1,164 @@
 const SSP_TITLE_ORIG = "Enterprise Logging and Auditing System Security Plan";
 const COMP_DEF_TITLE_ORIG = "Test Component Definition";
-const CATALOG_TITLE = "NIST Special Publication 800-53 Revision 4: Security and Privacy Controls for Federal Information Systems and Organizations";
+const CATALOG_TITLE =
+  "NIST Special Publication 800-53 Revision 4: Security and Privacy Controls for Federal Information Systems and Organizations";
 
-describe('Loading system security plans', () => {
-  it('loads SSPs by REST Mode', () => {
-    cy.navToSspEditor(SSP_TITLE_ORIG);
-    cy.waitForLoad();
-    cy.get(`[aria-label="show code"]`).click()
-    cy.scrollTo('bottom')
-    cy.contains('This is the control implementation for the system.').should('be.visible')
-  })
-  
-  it('loads system security plans by URL', () => {
-    cy.navToSspEditor(SSP_TITLE_ORIG)
-    cy.waitForLoad();
-    cy.contains("REST Mode").click()
-    cy.contains('OSCAL System Security Plan URL').first().should('exist').next().click().clear().type(Cypress.env('base_url') + "/oscal/v1/system-security-plans/cff8385f-108e-40a5-8f7a-82f3dc0eaba8")
-    cy.contains('Reload').click()
-    cy.contains(SSP_TITLE_ORIG).should('be.visible')
-    cy.scrollTo('bottom')
-    cy.contains('This is the control implementation for the system.').should('be.visible')
-  })
-  
-  it('displays Enterprise Asset Owners Party', () => {
+describe("Loading system security plans", () => {
```

Comment:
Moving forward, we need to just fix formatting on all these files in a separate pr. For now, **please do not start arbitrarily fixing style things**. It makes the diff a lot more confusing for no reason. Don't change it, but don't keep doing it.

Reply:
To be complete honest I typically use my style formatter to auto format files, which follows in line with our format on our other JS repos. Shouldn't we make file fixes when we touch a file, especially if we touch a large portion of the codebase? About half the tests in this file are related to `OSCALAnchorLinks`.

Reply:
Let's not refactor style in this repo until #164 is complete. That ticket will have a massive rework of the linting and style and anything we do until then may be wrong (IDE defaults can be weird).

In general, I like a fix-as-you-go; in `oscal-react-library` we're going to see a lit of diffs like this as TypeScript refactoring happens for example. So I am fine with doing it if there's a really good justification.

In this case, fixing style in a repo with no style rules may create a senseless back-and-forth as each editor does whatever it thinks is best. So if we want to do this, let's solve #164 and fix it all in one go.

@Bronstrom you do not have to revert the changes in order for this PR to be merged; however, let's refrain from doing this again at least until #164 is complete.